crates/astria-conductor/src/executor/mod.rs

```diff
@@ -501,32 +446,36 @@ impl Executor {
         &self,
     ) -> eyre::Result<tendermint::block::Height> {
         let Some(executable_block_height) = calculate_sequencer_block_height(
-            self.sequencer_height_with_first_rollup_block,
+            self.genesis.sequencer_genesis_block_number(),
             self.commitment_state.soft().number(),
         ) else {
             bail!(
                 "encountered overflow when calculating executable block height; sequencer height \
                  with first rollup block: {}, height recorded in soft commitment state: {}",
-                self.sequencer_height_with_first_rollup_block,
+                self.genesis.sequencer_genesis_block_number(),
                 self.commitment_state.soft().number(),
             );
         };
 
         Ok(executable_block_height.into())
     }
 
+    pub(crate) fn get_genesis(&self) -> GenesisInfo {
+        self.genesis.clone()
+    }
```

Comment:
```suggestion
    pub(crate) fn genesis(&self) -> &GenesisInfo {
        self.genesis
    }
```
accessors are conventionally named without `get_*` unless there is also a setter to avoid confusion.

Clone's should also be done explicitly by the caller of the method, not the method itself.

But either way - `GenesisInfo` can be `Copy`, so the cloning is not necessary